ads/appnexus.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  loadScript, writeScript,
+}
+from '../src/3p';
+
+const APPNEXUS_AST_URL = 'https://acdn.adnxs.com/ast/ast.js';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function appnexus(global, data) {
+  let useTtj = false;
+  const args = [];
+  args.push('size=' + data.width + 'x' + data.height);
+  if (data.tagid) {
+    args.push('id=' + encodeURIComponent(data.tagid));
+    writeScript(global, constructTtj(args));
+    usettj = true;
```

Comment:
This can be removed as a leaked global var -- `usettj` instead of `useTtj` -- and the var gets set in `construct Ttj()`